src/codex_query.py

```diff
@@ -140,68 +127,62 @@ def get_query(prompt_file):
 
     # get input from terminal or stdin
     if DEBUG_MODE:
-        entry = input("prompt: ") + '\n'
+        entry = input("prompt: ")
     else:
-        entry = sys.stdin.read()
+        # Remove extreaneous newlines and hashtag from the input string
```

Comment:
is this required by the dependency on prompt-engine, or is it a separate fix? could you expand the comment to explain why these chars are not allowed? By removing `\n` isn't there a risk of concatenating to words?

Reply:
note: if this is a requirement of prompt-engine, shouldn't this be done by the lib?

Reply:
In terminal, when the input is captured, it will capture the hashtag before the input, and the newline when you click enter. For example, you input "# whats my ip" and click enter, there will be a newline at the end, which is something we don't need. Prompt engine handles everything, so we only need whats my ip